README.md

```diff
@@ -12,12 +12,13 @@ CASR is maintained by:
 * [Andrey Fedotov](https://github.com/anfedotoff) \<fedotoff@ispras.ru\>
 * [Alexey Vishnyakov](https://github.com/SweetVishnya) \<vishnya@ispras.ru\>
 * [Georgy Savidov](https://github.com/Avgor46) \<avgor46@ispras.ru\>
+* [Ilya Yegorov](https://github.com/hkctkuy) \<Yegorov_Ilya@ispras.ru\>
 
 ## Overview
 
 CASR is a set of tools that allows you to collect crash reports in different
-ways. Use `casr-core` binary to deal with coredumps. Use `casr-san` to analyze ASAN
```

Comment:
Rebase problems?  casr-core is need to be here.

Reply:
Fixed